db/primary_migrate/20230309201053_add_sp_issuer_field_to_account_reset_request.rb

```diff
@@ -0,0 +1,5 @@
+class AddSpIssuerFieldToAccountResetRequest < ActiveRecord::Migration[7.0]
+  def change
+    add_column :account_reset_requests, :sp_issuer, :string
```

Comment:
typically we call this column `service_provider` or `issuer`, I have a slight preference for `issuer` and maybe if we wanted to be clear, we could call it `requesting_issuer` or something

Reply:
I like requesting_issuer, I added sp_issuer because it was in the ticket but I will change this to requesting_issuer.